subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/DeprecationCreationPerformanceTest.groovy

```diff
@@ -24,7 +24,7 @@ class DeprecationCreationPerformanceTest extends AbstractCrossVersionPerformance
         runner.testProject = "generateLotsOfDeprecationWarnings"
         runner.tasksToRun = ['help']
         runner.gradleOpts = ["-Xms1g", "-Xmx1g"]
-        runner.minimumBaseVersion = '4.9'
+        runner.minimumBaseVersion = '6.0'
```

Comment:
I think the `mininumBaseVersion` should be `6.3`.

Reply:
Hmmm...you're righ.